app/Dockerfile

```diff
@@ -1,17 +1,20 @@
 FROM alpine:3.9.6
 
-CMD ["python3"]
-
-WORKDIR /app
-
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
+ENV LISTEN_PORT 8000
+ENV WORKERS 1
 
+CMD ["sh", "-c", "gunicorn --bind 0.0.0.0:$LISTEN_PORT --workers $WORKERS --pythonpath app app.wsgi --timeout 300"]
+
+WORKDIR /app
 COPY . /app/
 
 RUN apk add -U bash python3 python3-dev libpq postgresql-dev unixodbc-dev musl-dev g++ libffi-dev \
   && pip3 install --upgrade pip setuptools \
   && pip3 install --no-cache-dir -r requirements.txt \
   && ln -s /usr/bin/python3 /usr/bin/python \
-  && apk del python3-dev postgresql-dev unixodbc-dev musl-dev g++ libffi-dev
+  && apk del python3-dev postgresql-dev unixodbc-dev musl-dev g++ libffi-dev \
+  && mkdir -p /app/staticfiles \
+  && chmod 777 /app/staticfiles
```

Comment:
Really need 777?

Reply:
Yes, as in prod we run these components as non-root user :)

```
backend_1   | PermissionError: [Errno 13] Permission denied: '/app/staticfiles/admin/css/vendor/select2/select2.min.d44571114a90.css'
```

Going with 777 is a bit too much, yes, but it should do for now.

```
backend_1   | Making staticfiles
```

Why is this even needed in a backend (granted, I didn't bother yet to dive into it)?